client/app/components/routes/operations/Page.tsx

```diff
@@ -3,14 +3,30 @@ import { Button } from "@mui/material";
 import { Suspense } from "react";
 import Operations from "@/app/components/routes/operations/Operations";
 import Loading from "@/app/components/loading/SkeletonGrid";
+import { actionHandler, getToken } from "@/app/utils/actions";
+
+export default async function OperationsPage() {
+  // 👤 Use NextAuth.js encrypted token to get authenticated user's user_guid
+  const token = await getToken();
+  const uid = token?.user_guid ?? "";
+
+  // 🚀 API call: Get operator id associated with this user_guid
+  const response = await actionHandler(
+    `registration/user-operator-id/${uid}`,
+    "GET",
+    "",
+  );
+  const operator_id = response?.operator_id ?? "";
 
-export default function OperationsPage() {
   return (
     <>
       <h1>Operations List</h1>
-      <Link href="/dashboard/operations/0/1">
-        <Button variant="contained">Add Operation</Button>
-      </Link>
+      {/* Conditionally render the button only if operator_id is an integer */}
+      {typeof operator_id === "number" && Number.isInteger(operator_id) && (
+        <Link href={`/dashboard/operations/${operator_id}/1`}>
```

Comment:
The operator placeholder is in line 120-121 in `/client/app/components/form/OperationsForm.tsx` (0 here is a placeholder for the operation ID, after creating an operation (clicking next on the first form step), the 0 is replaced by the operation id.)